internal/compose/compose.go

```diff
@@ -141,10 +143,18 @@ func NewProject(name string, paths ...string) (*Project, error) {
 	}
 
 	c := Project{
-		name,
-		paths,
+		name:             name,
+		dockerComposeV1:  true,
```

Comment:
nit: You could also skip this property here and set it only when `ve.Major` == 1 or undefined. But it would the same outcome :)

Reply:
Sorry I didn't quite get what you meant. Did you mean we could use the semver.Version (`*Version`) inside the `Project` and use `ver.Major()` in `ContainerName` instead ?

Reply:
Currently, you're setting the property `dockerComposeV1:  true` by default. But you could adjust it to be the other way around, so  just assume:

```go
c := Project{
   name: name,
   composeFilePaths: paths
}
```

and then just enable it for two cases:

```go
if ver.Major() == 1 {
   c.dockerComposeV1 = true
}
```

```go
	if err != nil {
		logger.Errorf("Unable to determine docker-compose version: %v. Defaulting to 1.x", err)
                 c.dockerComposeV1 = true
		return &c, nil
	}
```

Reply:
Let's see if I have permissions to push it here, so that we don't need to bother you anymore :)

Reply:
Done